pages/running_tests.md

```diff
@@ -97,6 +101,79 @@ For running Gatling tests on a microservice application, you have to:
 *   Run the microservice application
 *   Then, you can run Gatling tests
 
+### Using Maven/Gradle to run Gatling
+
+We do not generate Maven or Gradle configuration to run Gatling tests as this might cause some classpath issues with other plugins (mainly because of the use of Scale).
+Nevertheless you can leverage the offical [Maven](https://gatling.io/docs/current/extensions/maven_plugin/){:target="_blank" rel="noopener"}  or [Gradle](https://gatling.io/docs/current/extensions/gradle_plugin/){:target="_blank" rel="noopener"} plugin to execute the Gatling tests.
+
```

Comment:
I would put Maven plugin and Gradle plugin in the links instead of just Maven and Gradle.